easybuild/easyconfigs/m/magma/magma-2.7.0-GCC-12.2.0-CUDA-12.0.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'magma'
+version = '2.7.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://icl.cs.utk.edu/magma/'
+description = """The MAGMA project aims to develop a dense linear algebra library similar to
+ LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems."""
+
+toolchain = {'name': 'GCC', 'version': '12.2.0'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = ['https://icl.cs.utk.edu/projectsfiles/%(name)s/downloads/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['fda1cbc4607e77cacd8feb1c0f633c5826ba200a018f647f1c5436975b39fd18']
+
+#patches = ['%(name)s-%(version_major_minor)s.1_allow-all-sms.patch']
```

Comment:
This needs to be addressed. I got stuck on this as well, as they have made major changes, it's unclear whether it's needed at all anymore, and if so, needs to be completely rewritten to allow all SM's in the new built procedure.

Reply:
2.7.0 seems to find the correct GPU targets.
EB is setting this correctly as an argument to CMake: `-DGPU_TARGET="sm_61 sm_75" ` These are the values from my local config.cfg 

CMake output:
```
-- Found CUDA
--     CUDA_CUDART_LIBRARY: CUDA::cudart
--     compile for CUDA arch 6.1 (Pascal)
--     compile for CUDA arch 7.5 (Turing)
```

Reply:
good enough for me, i'll make sure it picks up my options when i test build later

Reply:
should remove the patches line?